PROJECT_NAME: "publish_path"

publish_path:
  distance_threshold: 0.1